<testsuites>
	<testsuite name="AllTests" tests="4">
		<testcase classname="AllTests" name="factorialTest"/>
		<testcase classname="AllTests" name="calcChangeTest">
			<failure message="../src/Test.cpp:26 calcChangeTest: &quot;5;&quot; == calcChange(5, numCoins, coinValues) expected:	5;	but was:		">
calcChangeTest: &quot;5;&quot; == calcChange(5, numCoins, coinValues) expected:	5;	but was:		
			</failure>
		</testcase>
		<testcase classname="AllTests" name="calcSumArrayTest">
			<failure message="../src/Test.cpp:47 calcSumArrayTest: &quot;1,4;9,1;11,2;18,1;22,0;&quot; == calcSum(sequence, 5) expected:	1,4;9,1;11,2;18,1;22,0;	but was:		">
calcSumArrayTest: &quot;1,4;9,1;11,2;18,1;22,0;&quot; == calcSum(sequence, 5) expected:	1,4;9,1;11,2;18,1;22,0;	but was:		
			</failure>
		</testcase>
		<testcase classname="AllTests" name="partitioningTest">
			<failure message="../src/Test.cpp:53 partitioningTest: 3025 == s_recursive(9,3) expected:	3025	but was:	1	">
partitioningTest: 3025 == s_recursive(9,3) expected:	3025	but was:	1	
			</failure>
		</testcase>
	</testsuite>
</testsuites>
